dace/transformation/interstate/if_extraction.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2019-2024 ETH Zurich and the DaCe authors. All rights reserved.
+""" If extraction transformation """
+
+from dace import data as dt, sdfg as sd, symbolic
+from dace.sdfg import utils as sdutil
+from dace.sdfg.state import SDFGState
+from dace.transformation import transformation
+from copy import deepcopy
+from dace.properties import make_properties
+
+
+def eliminate_branch(sdfg: sd.SDFG, initial_state: sd.SDFGState):
```

Comment:
Is there an analogous method in the sdfg? Should I make it one?

Reply:
I do not know of one, if this has potential applicability anywhere else, I would add a good docstring and put it on the `ControlFlowRegion` or SDFG Utilities.